src/main/java/com/lowcd/TCS/entity/Request.java

```diff
@@ -9,14 +10,16 @@
 @Table(name = "requests")
 @Data
 @NoArgsConstructor
+@AllArgsConstructor
 public class Request {
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
     @Column(name = "reqid")
     @Setter(value = AccessLevel.NONE)
     private Long reqId;
 
-    @ManyToMany(cascade = CascadeType.ALL)
+    @ManyToMany(cascade = {CascadeType.MERGE, CascadeType.PERSIST,CascadeType.REFRESH},
+            fetch = FetchType.LAZY)
```

Comment:
Confirm if the default value for fetch is `LAZY` then remove the `fetch` variable. applicable at multiple places.